metricflow/sql/render/redshift.py

```diff
@@ -10,6 +16,19 @@ def double_data_type(self) -> str:
         """Custom double data type for the Redshift engine"""
         return "DOUBLE PRECISION"
 
+    def visit_generate_uuid_expr(self, node: SqlGenerateUuidExpression) -> SqlExpressionRenderResult:  # noqa: D
+        """Generates a "good enough" random key to simulate a UUID.
+
+        NOTE: This is a temporary hacky solution as redshift does not have any UUID generation function.
+
+        Proposed solutions that requires more thinking:
+            - create a python UDF (Could we insert this without needing additional permissions?)
+        """
+        return SqlExpressionRenderResult(
+            sql="CONCAT(CAST(RANDOM()*100000 AS INT),CAST(RANDOM()*100000 AS INT),CAST(RANDOM()*100000 AS INT))",
```

Comment:
Seems like you could get ~16 significant digits (52 bits) out of RAND(). Thinking that since the UUIDs are 128-bit, we go with a similar number of bits?

Reply:
How does RANDOM work? I'd be worried about some weirdness in how they seed it by default.

Oftentimes what people do is concat the current timestamp in milliseconds to have a value as part of the sequence that is guaranteed to repeat only for a very small number of records.

I don't know if that's better or worse than three RANDOM() calls, so I don't think you need to change anything.